guides/tutorials/codelabs/pubsub-to-cf/index.md

```diff
@@ -4,10 +4,12 @@
  sidebar:
    nav: guides
 ---
-
- {% include alpha version="1.10 and later" %}
  
  {% include toc %}
+
+> This codelab assumes that you have enabled the `artifactsRewrite` feature flag. In `~/.hal/$DEPLOYMENT/profiles/settings-local.js` (where `$DEPLOYMENT` is typically `default`), add:
```

Comment:
As before, we'll need a way to make sure to remove this if the feature becomes the default and this flag becomes moot.

Reply:
I'm happy to go through in that case and at that time, search for the flag, and remove mentions like this. I only want it for now, while it's necessary in order to see the new UI.